gammapy/data/data_store.py

```diff
@@ -238,12 +227,31 @@ def get_observations(self, obs_id=None, skip_missing=False):
             Observation IDs (default of ``None`` means "all")
         skip_missing : bool, optional
             Skip missing observations, default: False
+        required_IRF : list of str
+            Runs will be added to the list of observations only if the
+            required IRFs are present. Otherwise, the given run will be skipped
+            Available options are:
+                * `aeff` : Effective area
+                * `bkg` : Background
+                * `edisp`: Energy dispersion
+                * `psf` : Point Spread Function
+            By default, all the IRFs are required.
 
         Returns
         -------
         observations : `~gammapy.data.Observations`
             Container holding a list of `~gammapy.data.Observation`
         """
+        available_IRF = ["aeff", "edisp", "psf", "bkg"]
```

Comment:
Please stick with the Python convention and make IRF all lower-case. like `required_irf` and `available_irf`